osschallenge/templates/osschallenge/profile.html

```diff
@@ -66,14 +75,27 @@ <h4 class="card-title">{{ profile.user.username }}</h4>
                 </tbody>
             </table>
             {% if request.user.id == profile.user.id %}
-
             <a href="/password_change">
                 <button class="btn btn-primary btn-profile">{% trans "Reset Password" %}</button>
             </a>
             <a href="/profile/edit">
                 <button class="btn btn-primary btn-profile">{% trans "Edit" %}</button>
             </a>
             {% endif %}
+            <div class="social-media-buttons">
```

Comment:
Also every text for the buttons should be translated

Reply:
Done